src/hooks/resources/release.ts

```diff
@@ -127,12 +131,87 @@ sbvrUtils.addPureHook('POST', 'resin', 'release', {
 	},
 });
 
+const checkNameValueUnique: sbvrUtils.Hooks = {
+	PRERUN: async (args) => {
+		const { request } = args;
+		const releaseIds = await getCurrentRequestAffectedIds(args);
+		if (releaseIds.length === 0) {
+			return;
+		}
+		const relIds = (await args.api.get({
+			resource: 'release',
+			options: {
+				$select: 'belongs_to__application',
+				$filter: {
+					id: { $in: releaseIds },
+				},
+			},
+		})) as AnyObject[];
+		if (
+			request.values.release_name !== undefined &&
+			request.values.release_name != null
```

Comment:
you can drop the `!== undefined` check, as `!= null` and `== null` will coerce undefined to null and behave correctly.

Reply:
done